### ISSUE TITLE
softmax

### ISSUE THREAD

**Issue by song6cy** [natanielruiz/deep-head-pose#118]
code/test_on_video_dlib.py:146: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  yaw_predicted = F.softmax(yaw)
code/test_on_video_dlib.py:147: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  pitch_predicted = F.softmax(pitch)
code/test_on_video_dlib.py:148: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  roll_predicted = F.softmax(roll)

dim is 0,1,or 2?
